pom.xml

```diff
@@ -20,7 +20,7 @@
 
   <groupId>com.yugabyte</groupId>
   <artifactId>native-protocol</artifactId>
-  <version>1.5.1-yb-1</version>
+  <version>1.5.1-yb-2-SNAPSHOT</version>
```

Comment:
When do you plan to change it to `1.5.1-yb-2`?

Reply:
May be this will be taken care of at the time of release. So this comment can be ignored.